javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,22 @@
+var $spans = document.querySelectorAll('span');
+var i = 0;
+
+function typingEvent(event) {
+  var $currentLetter = document.querySelector('.active');
+
+  if (!$currentLetter.nextElementSibling) {
+    if (i === 0) {
+      $spans[i].className = 'active';
+    }
+    return;
+  }
```

Comment:
Instead of querying the DOM again here, try just using your `$spans` and `i` to decide what to do.